### ISSUE TITLE
Notify member with card rule

### ISSUE THREAD

**Issue by ivan-paleo** [wekan/wekan#3304]
I haven't understood how to set up the rule to notify a member when he/she is added to a card.
I did create the rule, triggered by "when a member is added to a card", with action "send an email".
What I don't know is: 

- How do I specify the email address so that the email address of the member is added to the card?
- Ideally, there would be a link to the card, but it is apparently not possible yet (#3301).
- If I cannot link to the card, maybe I can at least mention the list and/or card title. How would I specify that?

In other words, what are the tags (or similar) that I can add in a notification email?

Maybe there is some specific documentation somewhere, but I looked for it and could not find anything detailed enough for my limited knowledge...

It is related to #3301; I'm not sure if you prefer a different issue or if you prefer a separate one.

**Comment by kairosdojo** [comment]
I'm interested in this too!
